strategies/affected/src/main/java/org/arquillian/smart/testing/strategies/affected/AffectedRunnerProperties.java

```diff
@@ -20,9 +25,23 @@
     static final String EXCLUSIONS = "exclusions";
 
     private final Properties properties = new Properties();
+    private AffectedConfiguration affectedConfiguration;
 
-    AffectedRunnerProperties(){
-        readFile(System.getProperty(SMART_TESTING_AFFECTED_CONFIG));
+    AffectedRunnerProperties(File rootDirectory) {
+        final Configuration configuration = Configuration.loadPrecalculated(rootDirectory);
+        final List<StrategyConfiguration> strategyConfigurations = configuration.getStrategiesConfiguration();
+        if (strategyConfigurations == null) {
+            readFile(null);
+            return;
+        }
+
+        final StrategyConfiguration affectedConfig = strategyConfigurations.stream()
```

Comment:
Maybe this logic could be moved inside `Configuration` object so you can do `configuration.getStrategyConfiguration("affected)"`